src/it/resources/org/checkstyle/suppressionxpathfilter/arraytypestyle/SuppressionXpathRegressionArrayTypeStyleMethodDef.java

```diff
@@ -0,0 +1,7 @@
+package org.checkstyle.suppressionxpathfilter.arraytypestyle;
+
+public class SuppressionXpathRegressionArrayTypeStyleMethodDef {
```

Comment:
All input files should start with `Input`.

Reply:
I think we can proceed.

Reply:
I'm new to Checkstyle. Indeed, when searching files in src/test/resources, many files starts with "Input". Maybe files in src/it/resources are not being checked in the same manner. I can rename the file, and same for other PRs. Also, rename existing files in the same package. Do you want me to keep prefix "SuppressionXpathRegression"?

1. `InputSuppressionXpathRegressionArrayTypeStyleMethodDef`
2. `InputArrayTypeStyleMethodDef`
3. Other if you have any suggestion 😀

Reply:
Let's keep going to follow format that we have, we can do massive renaming at the end in worse case.
As I mentioned in other PR, we need feedback from @timurt or need to search in whole history and find related discussion

Reply:
+1. Thanks

Reply:
@mincong-h You actually did a good thing by accidentally doing this. A veteran wouldn't have caught that we weren't validating this area.
Our UTs should have been checking their naming pattern but you brought to light it isn't.
https://github.com/checkstyle/checkstyle/blob/a73ff0890c27cbe8affc1a55f2710a231687bd85/src/test/java/com/puppycrawl/tools/checkstyle/internal/AllTestsTest.java#L45
I looked at it some and we don't have good input file names or their folder names too. It would require renaming all our files in that folder.

@romani I believe this was missed because of the google folders and we just skipped them. Though they should be named `Input` even in google, I am not sure we forced the folder to match the name of the test like we did in `test` folder.